icu4j/tools/build/src/main/java/com/ibm/icu/dev/tool/docs/APIStatusConsistencyChecker.java

```diff
@@ -64,6 +64,18 @@ public int errorCount() {
             {"clone",       "java.lang.Object()"},
     };
 
+    // Exceptions - API status of these methods are different from the parent
```

Comment:
The script checking API status makes the method inconsistency results more visible. These classes override parent object methods. My understanding is these are marked as deprecated by intent. Instead of leaving them as errors, I added them to the new exception list.